docker-compose.yml

```
@@ -70,6 +70,27 @@ services:
     networks:
       - monitoring_network
 
+  promtail:
+    image: grafana/promtail:latest
```

```suggestion
    image: grafana/promtail:3.2.1
```

Promtail is set to be deprecated, so we should ensure we don't use latest tag here :)